TradeItIosTicketSDK2/TradeItDeviceManager.swift

```diff
@@ -0,0 +1,69 @@
+import LocalAuthentication
+
+class TradeItDeviceManager {
+    
+    func authenticateUserWithTouchId(onSuccess onSuccess: () -> Void, onFailure: () -> Void) {
+        var error: NSError?
+        let myLocalizedReasonString = "Authentication is required"
+        
+        let context = LAContext()
+        
+        guard context.canEvaluatePolicy(LAPolicy.DeviceOwnerAuthentication, error: &error) else {
```

Comment:
DeviceOwnerAuthentication policy will show the touchId if there is a fingerprint registered otherwise it will show the passcode